src/content/en/updates/2018/07/nic68.md

```diff
@@ -0,0 +1,206 @@
+
+Note: Want the full list of changes? Check out the
+[Chromium source repository change list](https://chromium.googlesource.com/chromium/src/+log/67.0.3396.62..68.0.3440.74).
+
+
+## Add to Home Screen changes {: #a2hs }
+
+If your site meets the
+[add to home screen criteria](/web/fundamentals/app-install-banners/#criteria),
+Chrome will no longer show the add to home screen banner. Instead, you’re in
+control over when and how to prompt the user.
+
+To prompt the user, listen for the `beforeinstallprompt` event, then, save
+the event and add a button or other UI element to your app to indicate it can
+be installed.
+
+```javascript
+let installPromptEvent;
+
+window.addEventListener('beforeinstallprompt', (event) => {
+  // Prevent Chrome <= 67 from automatically showing the prompt
+  event.preventDefault();
+  // Stash the event so it can be triggered later.
+  installPromptEvent = event;
+  // Update the install UI to notify the user app can be installed
+  document.querySelector('#install-button').disabled = false;
+});
+```
+
+<img src="/web/updates/images/2018/06/a2hs-dialog-g.png" class="attempt-right">
+
+When the user clicks the install button, call `prompt()` on the saved
+`beforeinstallprompt` event, Chrome then shows the add to home screen dialog.
+
+<div class="clearfix"></div>
+
+```javascript
+btnInstall.addEventListener('click', () => {
+  // Update the install UI to remove the install button
+  document.querySelector('#install-button').disabled = true;
+  // Show the modal add to home screen dialog
+  installPromptEvent.prompt();
+  // Wait for the user to respond to the prompt
+  installPromptEvent.userChoice.then(handleInstall);
+});
+```
+
+<img src="/web/updates/images/2018/06/a2hs-infobar-g.png" class="attempt-right">
+
+To give you time to update your site Chrome will show a mini-infobar the first
+time a user visits a site that meets the add to home screen criteria. Once
+dismissed, the mini-infobar will not be shown again for a while.
+
+[**Changes to Add to Home Screen Behavior**](/web/updates/2018/06/a2hs-updates)
+has the full details, including code samples you can use and more.
+
+
+<div class="clearfix"></div>
+
+## Page Lifecycle API {: #page-lifecycle }
+
+<a href="/web/updates/images/2018/07/lifecycle-states.png">
+  <img src="/web/updates/images/2018/07/lifecycle-states.png" class="attempt-right">
+</a>
+
+When a user has a large number of tabs running, critical resources such as
+memory, CPU, battery and the network can be oversubscribed, leading to a
+bad user experience.
+
+If your site is running in the background, the system may suspend the it to
+conserve resources. With the new Page Lifecycle API, you can now listen for,
+and respond to these events.
+
+For example, if a tab needs to be discarded to conserve memory, the browser
+will fire the `frozen` event, where you can store any necessary state. Then,
+when the user refocuses the tab, the `resume` event is fired, making it
```

Comment:
The `resume` event is only fired if a page is frozen but *not* discarded.

Since this example kinda conflates the two concepts, I'd simplify the whole paragraph to this:

> For example, if a user's had a tab in the background for a while, the browser may choose to suspend script execution on that page to conserve resources. Before doing so, it will fire the `freeze` event, allowing you to close open IndexedDB or network connections or save any unsaved view state. Then, when the user refocuses the tab, the `resume` event is fired, where you can reinitialize anything that was torn down.